pkg/lib/machine-unknown-hostkey.html

```diff
@@ -16,9 +16,9 @@ <h4 class="modal-title" translate="yes">Unknown Host Key</h4>
 </div>
 
 <div class="modal-footer">
-    <button class="pf-c-button pf-m-secondary" data-dismiss="modal" translate="yes">Close</button>
-
     {{#key}}
         <button class="pf-c-button pf-m-primary" translate="yes">Connect</button>
     {{/key}}
+
+    <button class="pf-c-button pf-m-secondary" data-dismiss="modal" translate="yes">Close</button>
```

Comment:
Do we also maybe want to rename the buttons to close and make it a link like in all other dialogs?

Reply:
Good catch. Yeah, I guess we probably should.